src/shared/components/lock-screen/LockScreen.js

```diff
@@ -128,6 +132,9 @@ class LockScreen extends Component {
                 this.handleSubmission();
             }
             this.passwordInputRef.current.focus();
+            setTimeout(() => {
```

Comment:
Why do we need this? Isn't `input.addEventListener('blur', () => input.focus());` enough?

Reply:
Chrome was having some issues and launching a blur event at the end of the entire blur/focus flow when the user pressed `enter`, so I had to force a delayed focus to fix that.